### PR TITLE
🐛 Input validation bug fix

### LINKED ISSUE #11049: Input type number badInput validation will show valueMissing error message
## What's the issue?

Input type number badInput validation will show valueMissing error message in Firefix and Chrome.
Opera mobile is OK.

## How do we reproduce the issue?

Type "+" in number type input field, please refer below screeshot:
badinput-issue

## What browsers are affected?

Chrome, Firefox.

## Which AMP version is affected?

Powered by AMP ⚡ HTML – Version 1503095301111

### PR DESCRIPTION
**Pull request by yomeshgupta**
Fixes #11049 

**Problem**: Input type number badInput validation will show valueMissing error message in Firefox and Chrome.

**Solution**: In case of additional validation error along with **valueMissing**, additional error is given precedence.

### CODE REVIEW

**Comment by yomeshgupta** [comment]
CLA signed with both author ids.

**Comment by YomeshAdPushup** [comment]
I signed it!

**Comment by yomeshgupta** [comment]
I signed it!

File: extensions/amp-form/0.1/form-validators.js [hunk 1]
```diff
@@ -323,18 +305,16 @@ export class AbstractCustomValidator extends FormValidator {
...
-
   /** @override */
   report() {
     this.hideAllValidations();
     const inputs = this.inputs();
     for (let i = 0; i < inputs.length; i++) {
       if (!inputs[i].checkValidity()) {
         this.reportInput(inputs[i]);
-        inputs[i]./*REVIEW*/focus();
+        inputs[i]./*REVIEW*/ focus();
```
Comment (aghassemi) [inline extensions/amp-form/0.1/form-validators.js]: @yomeshgupta Could you please revert the unrelated formatting changes? Maybe your IDE automatically added them.

**Comment by aghassemi** [comment]
@yomeshgupta Thanks for the PR! there are commits from two different emails here (your personal and work ones). You would need to either sign the CLA with both emails or send another PR with commits from only the email that has signed the CLA.

**Comment by yomeshgupta** [comment]
@aghassemi By mistake, commits were pushed from two different accounts. To resolve that, I have signed CLA from both my accounts.

I did run gulp lint --fix to take care of formatting changes. Is there some specific formatting required for the AMP project?

**Comment by aghassemi** [comment]
@yomeshgupta We follow https://google.github.io/styleguide/jsguide.html for style and it is enforced by our Lint file.

Regarding CLA, you may need to close this PR and send another one where all commits are from the same email, unfortunately the CLA check bot will not allow merge until it is happy with the CLA

**Comment by yomeshgupta** [comment]
@aghassemi Okay. Thank you for the letting me know about the style guide. 

I am closing this PR and creating a new PR in which I have reverted the formatting changes and fixed dual account issue and input validation issue. 

New PR - https://github.com/ampproject/amphtml/pull/18069
